src/app/containers/StoryPromo/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { Fragment } from 'react';
+import { shape } from 'prop-types';
+import { latin } from '@bbc/gel-foundations/scripts';
+import StoryPromoComponent, {
```

Comment:
Just being nosey, should this be `StoryPromo` instead of `StoryPromoComponent`? I believe it is defined and exported like that in `Psammead`.

Reply:
@DenisBBC As its the default export it can be renamed, I did it like this as it clashes with the `StoryPromo` that this file exports.

We could do 
`StoryPromoComponent` and export `StoryPromo`
or
`StoryPromo` and export `StoryPromoContainer` 

Idm :)

Reply:
I thought that, but I just wanted a confirmation :)
I am happy as it is then 👍